test/integration/apiserver.go

```diff
@@ -0,0 +1,359 @@
+// Copyright 2017 Istio Authors
```

Comment:
also maybe put it under `test/integration/kube/apiserver_test.go` because it is kube integration